eos-updater/poll-common.c

```diff
@@ -514,8 +514,9 @@ should_follow_checkpoint (OstreeSysroot     *sysroot,
   g_autoptr(GHashTable) hw_descriptors = NULL;
   const gchar *sys_vendor, *product_name;
   gboolean is_eos3a_to_eos4 =
```

Comment:
It would also be useful to add a comment above this variable like:
```
/* https://phabricator.endlessm.com/T32542, https://phabricator.endlessm.com/T32552 */
```
to allow the reasoning for the choice of branch names/prefixes to be looked up later easily.

Reply:
good point!  Thanks